libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Network/Network/Action/CSFAction.m

```diff
@@ -97,12 +97,17 @@ - (NSURL*)urlForActionWithError:(NSError**)error {
         }
         return nil;
     }
-    
-    if ([baseURL.absoluteString hasSuffix:@"/"] && [path hasPrefix:@"/"]) {
+
+    // 'baseUrl' needs to account for URLs with extra trailing slashes, like custom community URLs.
+    NSMutableString *baseUrlString = [NSMutableString stringWithString:baseURL.absoluteString];
+    if (![baseUrlString hasSuffix:@"/"]) {
+        [baseUrlString appendString:@"/"];
+    }
+    if ([path hasPrefix:@"/"]) {
         [path deleteCharactersInRange:NSMakeRange(0, 1)];
```

Comment:
Yeah - is this a bad merge/rebase? @trooper2013 Let's make sure this accurately represents the latest changes in this file.